internal/pkg/describe/service.go

```diff
@@ -475,6 +475,26 @@ func (e containerEnvVars) humanString(w io.Writer) {
 	printTable(w, headers, rows)
 }
 
+// envSecret contains secrets for an rdws service.
+type envSecret struct {
```

Comment:
Can we rename this struct to `rdwsSecret` since it's really similar to the existing `secret` type which also has an "environment" field:
https://github.com/aws/copilot-cli/blob/877347fa4f8f1112d7f5b7cb18c6726b3aa2d8e4/internal/pkg/describe/service.go#L478-L485

Reply:
Done!